cpp/src/sampling/detail/prepare_next_frontier_impl.cuh

```diff
@@ -58,11 +58,12 @@ prepare_next_frontier(
   bool dedupe_sources,
   bool do_expensive_check)
 {
+  // Create a single new frontier.
   vertex_partition_device_view_t<vertex_t, multi_gpu> d_vertex_partition(vertex_partition);
 
-  size_t frontier_size = sampled_dst_vertices.size();
+  size_t frontier_size = sampled_dst_vertices.size(); // std:reduce for all edge typw within hop / Reduce the size of each of the span. std::reduce for each of the size - example code that concatenate device vectors (look at that)
```

Comment:
`all edge typew` => `all edge_types`

But not sure we need this comment here. This function doesn't need to know whether `sampled_dst_vertices` is from a single vector or multiple vectors.

Reply:
These changes weren't meant to be committed. I will undo all changes to this file as I didn't modify anything here